pl_bolts/utils/semi_supervised.py

```diff
@@ -24,46 +26,46 @@ class Identity(torch.nn.Module):
         model.fc = Identity()
 
     """
-    def __init__(self):
+    def __init__(self) -> None:
         super(Identity, self).__init__()
 
-    def forward(self, x):
+    def forward(self, x: Tensor) -> Tensor:
         return x
 
 
-def balance_classes(X: np.ndarray, Y: list, batch_size: int):
+def balance_classes(X: np.ndarray, labels: Sequence[int], batch_size: int) -> Tuple[np.ndarray, np.ndarray]:
```

Comment:
Ah, I did not think it can be `torch.Tensor` or `numpy.ndarray`, I think this will solve the problem.
> (or you think of other types, too?).
It can also be `Tuple`, hence I kept it as `Sequence`